src/components/timeline/index.js

```diff
@@ -257,11 +206,6 @@ class Timeline extends React.PureComponent<Props, State> {
             hiddenTrackCount={hiddenTrackCount}
             changeRightClickedTrack={changeRightClickedTrack}
           />
-          <TimelineSettingsActiveTabView
```

Comment:
Instead of removing this, would it be better to do something like this?

```js
true ? null : <TimelineSettingsActiveTabView
  activeBrowsingContextID={activeBrowsingContextID}
  showTabOnly={showTabOnly}
  changeShowTabOnly={changeShowTabOnly}
/>
```

My reasoning here is that we'll want this feature back, and it would help this code not to bitrot as it would still be type checked. We could skip the test for now as well.

This is totally optional, and happy if you just want to remove it too.

Reply:
Yeah, that sounds better to me now too. My reasoning was to remove it so we can actually revert this commit later and bring it back easily, but then it's gonna be vulnerable to getting bitrot (and we can just revert other commit as well, so my reasoning doesn't make sense 😄). Will keep the code and just skip the tests as you recommended.